test/index.test.js

```diff
@@ -44,13 +47,11 @@ describe('Rollbar', function() {
       it('should create the window.Rollbar object', function() {
         analytics.assert(!window.Rollbar);
         analytics.initialize();
-        analytics.page();
         analytics.assert(window.Rollbar);
       });
 
       it('should have all of the correct methods', function() {
         analytics.initialize();
-        analytics.page();
```

Comment:
These weren't actually doing anything. I think I originally copied them from some other integration's tests.

E.g. We don't need `page()` to be called in order to verify that the Rollbar notifier has all of the appropriate methods after initialization.